src/mnode/rpc/masternode.cpp

```diff
@@ -146,16 +150,15 @@ Available modes:
             {
                 case RPC_CMD_MNLIST::activeseconds:
                 {
-                    if (!strFilter.empty() && strOutpoint.find(strFilter) == string::npos)
+                    if (is_filtered(strOutpoint))
                         continue;
                     obj.pushKV(strOutpoint, (int64_t)(mn.lastPing.sigTime - mn.sigTime));
```

Comment:
BTW, can use obj.pushKV(move(strOutpoint), ....) in all cases

Reply:
Ok.  That seems outside the scope of this issue, but I can make that change.